docs/workflows/common-workflow-steps/custom-script.md

```diff
@@ -227,87 +227,153 @@ sudo -E printenv
 
 ### How can I send a custom Email?
 
-Appcircle provides a ready-to-use email structure in the [**Testing Distribution**](/testing-distribution/create-or-select-a-distribution-profile#share-binary), and [**Publish**](/publish-integrations/common-publish-integrations/get-approval-via-email) modules. This structure varies across the three modules. If desired, the user can customize this structure by using the custom script below to send their own custom email.
+Appcircle provides a **ready-to-use email** structure in the [**Testing Distribution**](/testing-distribution/create-or-select-a-distribution-profile#share-binary), and [**Publish**](/publish-integrations/common-publish-integrations/get-approval-via-email) modules. This structure varies across the two modules. If desired, the user can customize this structure by using the [**Custom Script**](/workflows/common-workflow-steps/custom-script) below to send their own custom email.
+
+The following Bash script is set to use a **Gmail SMTP Server**. For more information, please visit [**Gmail SMTP Server**](https://support.google.com/a/answer/176600?hl=en) documentation. 
 
 ```bash
 
+# Set SMTP server 
+CS_HOST="smtp.gmail.com"
+CS_PORT="587"
+CS_ACCOUNT="gmail"
+
+# Make sure to replace CS_EMAIL, CS_USERNAME, and CS_PASSWORD with your account details
+CS_EMAIL="your-email-address@gmail.com"
+CS_USERNAME="your-email-address@gmail.com"
+CS_PASSWORD="your-email-password"
+
+# Set email details
+CS_EMAIL_SUBJECT="Test Email Subject"
+CS_EMAIL_TO="reciever-address@mail.com,reciever-2-address@mail.com"
```

Comment:
We should change the concept from `reciever` to `recipient` since that's the common usage in the email domain. [ref](https://www.google.com/search?q=reciever+email+address)

You can see the list of changes below.

![Screenshot 2024-11-10 at 08 04 32](https://github.com/user-attachments/assets/69e061f7-45fc-497e-bd11-c5dd093a3024)